core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -1017,7 +1017,7 @@ object ConsumerGroupCommand extends Logging {
         CommandLineUtils.checkRequiredArgs(parser, options, bootstrapServerOpt)
 
         if (options.has(newConsumerOpt)) {
-          Console.err.println(s"The $newConsumerOpt option is deprecated and will be removed in a future major release." +
+          Console.err.println(s"The $newConsumerOpt option is deprecated and will be removed in a future major release. " +
```

Comment:
nit: Perhaps for consistency with the option messages, we can let the warning mention `--new-consumer` directly? The message reads a bit oddly with `[new-consumer]` instead. Say for the bootstrap server below.

Reply:
Yes, I agree. I changed that.